api/backup/backup.go

```diff
@@ -80,7 +80,8 @@ func CreateBackupArchive(password string, gate *offlinegate.OfflineGate, datasto
 }
 
 func backupDb(backupDirPath string, datastore dataservices.DataStore) error {
-	backupWriter, err := os.Create(filepath.Join(backupDirPath, "portainer.db"))
+	dbFileName := datastore.Connection().GetDatabaseFileName()
```

Comment:
It's a good idea, but I'm afraid we'd need tweak the restore as well as it has `var filesToRestore = append(filesToBackup, "portainer.db")` meaning the db name is hardcoded.